daemon/src/db.rs

```diff
@@ -497,7 +497,7 @@ mod tests {
                 transition_timestamp: SystemTime::now(),
             },
         };
-        insert_new_cfd_state_by_order_id(cfd.order.id, cfd.state, &mut conn)
+        insert_new_cfd_state_by_order_id(cfd.order.id, cfd.state.clone(), &mut conn)
```

Comment:
This should be squashed into some other commit, whichever one caused it to fail. You can find out with `git rebase master -x 'cargo test --workspace'`